.gitignore

```diff
@@ -0,0 +1,3 @@
+
+
+.idea/
```

Comment:
Drop this file please, this seems like something that should be in one's global .gitignore.